citesphere/src/main/java/edu/asu/diging/citesphere/core/search/model/impl/Reference.java

```diff
@@ -10,7 +10,7 @@
 
 import edu.asu.diging.citesphere.model.bib.ItemType;
 
-@Document(indexName="#{@indexName}", type = "_doc")
+@Document(indexName="#{@indexName}", type = "_doc", createIndex = false)
```

Comment:
why has this been changed? It should automatically create an index.

Reply:
The reason for changing the createIndex attribute to false is to ensure that the Citesphere service starts successfully even if Elasticsearch is not running. When createIndex is set to true (or omitted, since the default is true), the service attempts to create an index upon startup. If Elasticsearch is not running, this operation fails, preventing the Citesphere service from starting.

By setting createIndex to false, we're essentially allowing the service to start without attempting to create the index.

Reply:
what are the exception try/catch blocks for then?

Reply:
I tested without try catch block, it worked properly, removed try catch block in another pull request.